docs/release_notes/v0.9.1-alpha.md

```diff
@@ -0,0 +1,30 @@
+The v0.9.1 release adds `flightsql` data connector and `sqlite` data store. Improved the usability by adding more `spice` CLI commands.
+
+## Highlights in v0.9.1-alpha
+
+**FlightSQL data connector**: [Arrow Flight SQL](https://arrow.apache.org/docs/format/FlightSql.html) can now be used as a connector for federated SQL query.
+
+**SQlite data backend**: [SQLite](https://www.sqlite.org/) can now be used as a data store for acceleration.
+
+## Contributors
+
+- @phillipleblanc
+- @mitchdevenport
+- @Jeadie
+- @ewgenius
+- @sgrebnov
+- @y-f-u
+- @lukekim
+
+## New in this release
+
+- **Adds** FlightSQL data connector (`flightsql`).
+- **Adds** SQLite data store (`sqlite`).
```

Comment:
Might want to mention support for in-memory and file based since not all data stores support both